app/src/header/tabs.rs

```diff
@@ -19,11 +22,33 @@ impl<'a> Widget for Tabs<'a> {
 			tabs
 				.iter()
 				.enumerate()
-				.map(|(i, _)| {
+				.map(|(i, tab)| {
+					let mut text = format!(" {} ", i + 1);
+					if let Some(dir_name) = tab.current_name() {
+						text.push_str(dir_name);
+					}
+
+					let threshold = THEME.tab.max_width.max(3) - 1;
```

Comment:
It seems that `threshold` doesn't need to contain 2 spaces, as the spaces are `padding`, not `content`.

Currently, the spaces should always be present, and the user can be allowed to configure their own padding characters later on.